draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2235,11 +2455,11 @@ or TI-LFA (MPLS based SR-TE case) through a protection port. At
 the same time MDSC, after checking that optical network connection
 is still fine, would trigger the reconfiguration of the back-up
 port of the IP router and of the muxponder to re-use the same
-WDM tunnel as the one used originally for the failed IP router port. Once
+DWDM tunnel as the one used originally for the failed IP router port. Once
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
WDM tunnel as the one used originally for the failed IP router port. Once
```